docs/examples/3_crude_from_saudi_arabia_to_india.py

```diff
@@ -15,31 +15,32 @@
     now = datetime.utcnow()
     one_month_ago = now - relativedelta(months=1)
 
+    # For this analysis we need the geography ID for India, and the geography ID for Saudi Arabia. We're going to
+    # show 2 ways to retrieve geography IDs. You'll want to chose method 1 or 2 depending on your use case.
+
+    # Option 1. We look up a geography with an exact matching name
+    saudi_arabia = Geographies().search("Saudi Arabia", exact_term_match=True).to_list()[0].id
+
+    # Option 2. We search for geographies with similar names, then pick the one we're looking for
+
     # First we find the ID for the country India. Note that when searching geographies with the term 'india', we'll
     # retrieve all geographies with india in the name, ie Indiana, British Indian Ocean Territory...
     all_geogs_with_india_in_the_name = Geographies().search("india").to_list()
 
+    # If running interactively, you may want to print all the names here to inspect them for yourself
+    for g in all_geogs_with_india_in_the_name:
+        print(g.name)
+
     # We're only interested in the country India here
     india = [
         g.id for g in all_geogs_with_india_in_the_name if g.name == "India"
     ]
```

Comment:
Should we use the layer option here? I.e. `if g.layer == "country"`

Reply:
We could do - I wanted to highlight here that that this is the second way to filter on exactly the name - by doing it manually yourself. I didn't mean the 'we want the country India here' comment to focus to much that the plolygon is a country layer.

Reply:
Fair enough